[2023-08-22T05:00:16.765+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_python_operator.virtualenv_classic manual__2023-08-22T05:00:02.466058+00:00 [queued]>
[2023-08-22T05:00:16.775+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_python_operator.virtualenv_classic manual__2023-08-22T05:00:02.466058+00:00 [queued]>
[2023-08-22T05:00:16.776+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-22T05:00:16.790+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonVirtualenvOperator): virtualenv_classic> on 2023-08-22 05:00:02.466058+00:00
[2023-08-22T05:00:16.796+0000] {standard_task_runner.py:57} INFO - Started process 4363 to run task
[2023-08-22T05:00:16.799+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_python_operator', 'virtualenv_classic', 'manual__2023-08-22T05:00:02.466058+00:00', '--job-id', '14', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_python_operator.py', '--cfg-path', '/tmp/tmpthb95y86']
[2023-08-22T05:00:16.802+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask virtualenv_classic
[2023-08-22T05:00:16.854+0000] {task_command.py:410} INFO - Running <TaskInstance: example_python_operator.virtualenv_classic manual__2023-08-22T05:00:02.466058+00:00 [running]> on host 908f67153615
[2023-08-22T05:00:16.936+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_python_operator' AIRFLOW_CTX_TASK_ID='virtualenv_classic' AIRFLOW_CTX_EXECUTION_DATE='2023-08-22T05:00:02.466058+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-22T05:00:02.466058+00:00'
[2023-08-22T05:00:16.938+0000] {process_utils.py:181} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venv0pyd2s4h --system-site-packages
[2023-08-22T05:00:16.946+0000] {process_utils.py:185} INFO - Output:
[2023-08-22T05:00:17.994+0000] {process_utils.py:189} INFO - created virtual environment CPython3.7.17.final.0-64 in 322ms
[2023-08-22T05:00:17.995+0000] {process_utils.py:189} INFO -   creator CPython3Posix(dest=/tmp/venv0pyd2s4h, clear=False, no_vcs_ignore=False, global=True)
[2023-08-22T05:00:17.995+0000] {process_utils.py:189} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-08-22T05:00:17.995+0000] {process_utils.py:189} INFO -     added seed packages: pip==23.1, setuptools==67.6.1, wheel==0.40.0
[2023-08-22T05:00:17.995+0000] {process_utils.py:189} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-08-22T05:00:18.036+0000] {process_utils.py:181} INFO - Executing cmd: /tmp/venv0pyd2s4h/bin/pip install -r /tmp/venv0pyd2s4h/requirements.txt
[2023-08-22T05:00:18.045+0000] {process_utils.py:185} INFO - Output:
[2023-08-22T05:00:19.739+0000] {process_utils.py:189} INFO - Collecting colorama==0.4.0 (from -r /tmp/venv0pyd2s4h/requirements.txt (line 1))
[2023-08-22T05:00:19.741+0000] {process_utils.py:189} INFO -   Using cached colorama-0.4.0-py2.py3-none-any.whl (21 kB)
[2023-08-22T05:00:25.171+0000] {process_utils.py:189} INFO - Installing collected packages: colorama
[2023-08-22T05:00:25.172+0000] {process_utils.py:189} INFO -   Attempting uninstall: colorama
[2023-08-22T05:00:25.173+0000] {process_utils.py:189} INFO -     Found existing installation: colorama 0.4.6
[2023-08-22T05:00:25.173+0000] {process_utils.py:189} INFO -     Not uninstalling colorama at /home/***/.local/lib/python3.7/site-packages, outside environment /tmp/venv0pyd2s4h
[2023-08-22T05:00:25.174+0000] {process_utils.py:189} INFO -     Can't uninstall 'colorama'. No files were found to uninstall.
[2023-08-22T05:00:25.190+0000] {process_utils.py:189} INFO - Successfully installed colorama-0.4.0
[2023-08-22T05:00:25.435+0000] {process_utils.py:189} INFO - 
[2023-08-22T05:00:25.435+0000] {process_utils.py:189} INFO - [notice] A new release of pip is available: 23.1 -> 23.2.1
[2023-08-22T05:00:25.435+0000] {process_utils.py:189} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-08-22T05:00:25.558+0000] {process_utils.py:181} INFO - Executing cmd: /tmp/venv0pyd2s4h/bin/python /tmp/venv0pyd2s4h/script.py /tmp/venv0pyd2s4h/script.in /tmp/venv0pyd2s4h/script.out /tmp/venv0pyd2s4h/string_args.txt
[2023-08-22T05:00:25.566+0000] {process_utils.py:185} INFO - Output:
[2023-08-22T05:00:26.202+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-08-22T05:00:26.211+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=example_python_operator, task_id=virtualenv_classic, execution_date=20230822T050002, start_date=20230822T050016, end_date=20230822T050026
[2023-08-22T05:00:26.233+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-22T05:00:26.246+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
